stdlib/public/core/Builtin.swift

```diff
@@ -691,6 +691,18 @@ func _isPOD<T>(_ type: T.Type) -> Bool {
   return Bool(Builtin.ispod(type))
 }
 
+/// Returns `true` if `type` is known to refer to a concrete type once all
+/// optimizations and constant folding has occurred at the call site. Otherwise,
+/// this returns `false` if the check has failed.
+///
+/// Note that there may be cases in which, despite `T` being concrete at some
+/// point in the caller chain, this function will return `false`.
+@_alwaysEmitIntoClient
```

Comment:
@stephentyrone it seems that the `@_alwaysEmitIntoClient` attribute is causing `_isConcrete` to fail to return `true`. `@_transparent` seems to work so I'll revert it to that. Would it be redundant to keep `@ _alwaysEmitIntoClient` at that point?

Reply:
No, you'll need both. `@_transparent` gets you the optimization you want, `@_aEIC` saves you from needing availability.